docs/user/TerraformGCPAuthentication.md

```diff
@@ -96,7 +96,7 @@ roles/resourcemanager.projectIamAdmin
 
 ## Create the Service Account Keyfile
 
-When running terraform on a workstation outside of the Google Cloud Platform, you persist the Service Account information to a JSON file, and then [specify that file when running terraform](#Terraform-project-variables-to-authenticate-with-GCP).
+When running terraform on a workstation outside of the Google Cloud, you persist the Service Account information to a JSON file, and then [specify that file when running terraform](#Terraform-project-variables-to-authenticate-with-GCP).
```

Comment:
"outside of Google Cloud, you store the Service Account information in a JSON file, and then [specify that file when running Terraform]"